packages/cli/bin/n8n

```diff
@@ -25,10 +21,18 @@ if (process.argv.length === 2) {
 var nodeVersion = process.versions.node.split('.');
 
 if (parseInt(nodeVersion[0], 10) < 14) {
-	console.log(`\nYour Node.js version (${process.versions.node}) is too old to run n8n.\nPlease update at least to Node.js v14 or to the recommended Node.js v16!\n`);
+	console.log(`
```

Comment:
Should we set 18 as upper limit?

Reply:
done. now we explicitly check for node 14, 16, or 18. 
15, 17 & 19 aren't supported.